_observing-your-data/alerting/comp-monitors.md

```diff
@@ -0,0 +1,633 @@
+
+* AND = `&&`
+* OR = `||`
+* NOT = `!`
+* Precedence = `()`
+
+See the following examples to understand how each is used in the monitor definition.
+
+* **Example 1**
+   
+   `monitor[id=1] && monitor[id=2]`
+   
+   The following conditions for delegate monitors will trigger the composite monitor to produce an alert when both monitor #1 AND monitor #2 generate an alert.
+
+* **Example 2**
+   
+   `monitor[id=1] || monitor[id=2]`
+
+   The following conditions will trigger the composite monitor to produce an alert when either monitor #1 OR monitor #2 generates an alert.
+
+* **Example 3**
+   
+   `monitor[id=1] && (!monitor[id=2] || monitor[id=3])`
+
+   The following conditions will trigger the composite monitor to produce an alert when monitor #1 generates an alert AND monitor #2 does NOT, OR, monitor #3 does generate an alert.
+   
+The order of monitor IDs in the Painless script does not define the sequence of execution for the monitors. The sequence of monitor execution is defined in the `inputs.composite_input.sequence.delegates.order` field in the request.
+{: .note }
+
+
+### Get Composite Monitor
+
+This API retrieves information on the specified monitor.
+
+```json
+GET _plugins/_alerting/workflows/<workflow_id>
+```
+{% include copy-curl.html %}
+
+#### Path parameters
+
+| Field | Type | Description |
+| :--- | :--- | :--- |
+| `workflow_id` | String | The composite monitor's [workflow Id](#key-terms). |
+
+
+### Update Composite Monitor
+
+This API updates the composite monitor's details. See [Create Composite Monitor](#create-composite-monitor) for description of request fields.
+
+```json
+PUT _plugins/_alerting/workflows/<workflow_id>
+{
+    "owner": "security_analytics",
+    "type": "workflow",
+    "schedule": {
+        "period": {
+        "interval": 1,
+        "unit": "MINUTES"
+        }
+    },
+    "inputs": [
+        {
+            "composite_input": {
+                "sequence": {
+                    "delegates": [
+                        {
+                            "order": 1,
+                            "monitor_id": "grsbCIcBvEHfkjWFeCqb"
+                        },
+                        {
+                            "order": 1,
+                            "monitor_id": "agasbCIcBvEHfkjWFeCqa"
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "enabled_time": 1679468231835,
+    "enabled": true,
+    "workflow_type": "composite",
+ 
```

Comment:
we can add chained alert triggers field from create api example here too. Judgement call.